src/Capgemini.PowerApps.PackageDeployerTemplate/Constants.cs

```diff
@@ -349,5 +354,64 @@ public static class Fields
                 public const string ConnectionReferenceLogicalName = "connectionreferencelogicalname";
             }
         }
+
+        /// <summary>
+        /// Constants related to the mailbox.
+        /// </summary>
+        public static class Mailbox
+        {
+            /// <summary>
+            /// The logical name.
+            /// </summary>
+            public const string LogicalName = "mailbox";
+
+            /// <summary>
+            /// Field logical names.
+            /// </summary>
+            public static class Fields
+            {
+                /// <summary>
+                /// The mailbox regarding object ID.
+                /// </summary>
+                public const string RegardingObjectid = "regardingobjectid";
+
+                /// <summary>
+                /// The Test Email Configuration Scheduled.
+                /// </summary>
+                public const string TestEmailConfigurationScheduled = "testemailconfigurationscheduled";
+
+                /// <summary>
+                /// The Mailbox Status.
+                /// </summary>
+                public const string MailboxStatus = "mailboxstatus";
+            }
+
+            /// <summary>
+            /// Constants related to the queue entity.
+            /// </summary>
+            public static class Queue
```

Comment:
This has been accidentally embedded within the `Mailbox` static class

Reply:
Good spot!